### PR TITLE
Add timeout options to zabbix_settings

### PR DESCRIPTION
**Pull request by aplathrop**
##### SUMMARY
Zabbix 7.0 added several new settings for timeouts (timeout_zabbix_agent, timeout_simple_check, etc) and these are currently missing in the zabbix_settings module. This PR adds them in

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- zabbix_settings

##### ADDITIONAL INFORMATION
An example of using one of the timeouts is below
```
- community.zabbix.zabbix_settings:
    timeout_zabbix_agent: 3s
```

### CODE REVIEW

**Comment by masa-orca** [comment]
Please git pull --rebase origin/main.

File: plugins/modules/zabbix_settings.py [hunk 1]
```diff
@@ -800,6 +882,56 @@ def update_settings(
                         "iframe_sandboxing_exceptions"
                     ] = iframe_sandboxing_exceptions
 
+            if isinstance(timeout_zabbix_agent, str):
```
Comment (masa-orca) [inline plugins/modules/zabbix_settings.py:885]: Please check zabbix version before dealing with the parameters.
e.g.
```
        if LooseVersion(host._zbx_api_version) >= LooseVersion("7.0"):
            # If monitored_by and proxy_group are not specified as a module parameters, use the existing setting
            if monitored_by is None:
                monitored_by = int(zabbix_host_obj["monitored_by"])
            if proxy_group is None:
                proxy_group_id = int(zabbix_host_obj["proxy_groupid"])

```

Comment (aplathrop) [reply plugins/modules/zabbix_settings.py]: This should be added now

**Review by masa-orca — APPROVED** [review]
LGTM!

**Comment by masa-orca** [comment]
I approved this PR. however, please test all parameters for 6.4 or lower if you can .

**Comment by aplathrop** [comment]
I've added the rest of the new parameters to the <7.0 tests

**Comment by masa-orca** [comment]
Sorry. I want you to check these patterns.
```
- name: test - timeout_zabbix_agent parameter fail for older version of Zabbix
  community.zabbix.zabbix_settings:
    timeout_zabbix_agent: 3s

- name: test - timeout_simple_check parameter fail for older version of Zabbix
  community.zabbix.zabbix_settings:
    timeout_simple_check: 30s
    
 - name: test - timeout_snmp_agent parameter fail for older version of Zabbix
  community.zabbix.zabbix_settings:
    timeout_snmp_agent: 30s
....
```
because your tests cannot check all errors.
Warnings about this: https://github.com/ansible-collections/community.zabbix/pull/1427/checks?check_run_id=34134135951

**Comment by aplathrop** [comment]
Okay, there should be a test for each of the new parameters

**Comment by masa-orca** [comment]
Thank you to update tests!

**Pull request merged**